src/html/partials/header.html

```diff
@@ -0,0 +1,14 @@
+<header class="page__header">
```

Comment:
Similar BEM observation here, since there isn't a `page` class at the top, you don't necessarily have to have `page__header` here, it could just be `page-header` or just `header` (for simplicity).